tests/fuzzer/fuzzer.py

```diff
@@ -25,7 +25,7 @@ class FailedTest(Exception):
 
 
 class ErrorType(str, Enum):
-    PATHFINDING = "PATHFINDING",
+    PATHFINDING = ("PATHFINDING",)
```

Comment:
I'm not really sure what this changes, but it bothers me a bit that it's not consistent anymore, can it be changed for the other values as well?